### DISCUSSION TITLE
No reconnect option

### DISCUSSION THREAD

**Discussion by aqildad-create** [pschatzmann/arduino-audio-tools#924] [Q&A] [answered]
need to reconnect A2DPStream .  their is no option right ? 

their is way in ESP32-A2DP like this a2dp_sink.set_auto_reconnect(true); but not in A2DPStream  right ?

![image](https://github.com/pschatzmann/arduino-audio-tools/assets/65456998/6538560d-542a-4bbb-980d-aa9521bccc47)

**Comment by pschatzmann** [comment, marked as answer]
It seems that you forgot to look at the A2DPConfig!
